deploy/Chart/templates/dashboard/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.dashboard.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dashboard-role
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: dashboard-role
+    app.kubernetes.io/part-of: radius
+rules:
+  - apiGroups: ['', 'api.ucp.dev']
+    resources: ['*']
+    verbs: ['*']
```

Comment:
Do we want to limit the dashboard to just GET for now, since the dashboard shouldn't do any write operations? Will help with a least-privilege security model.

Reply:
sure, will update